src/MicrosoftTeams.ts

```diff
@@ -21,15 +31,45 @@ namespace microsoftTeams {
 
   const version = "1.2";
 
+  // if schema is not specified then default use https:// only http/https schema are supported
   const validOrigins = [
     "https://teams.microsoft.com",
     "https://teams.microsoft.us",
     "https://int.teams.microsoft.com",
     "https://devspaces.skype.com",
     "https://ssauth.skype.com",
-    "http://dev.local" // local development
+    "http://dev.local", // local development
+    "https://msft.spoppe.com",
+    "*.sharepoint.com",
```

Comment:
The mix of explicit origins and wildcarded hostnames makes this hard to maintain-- someone will be tempted to add an entry like `https://*.teams.microsoft.com` which will not work as expected.